plugins/migration/bin/rename.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A script that automatically upgrades pieces of the Blockly API
+ *     that have been renamed by modifying developers' local files.
+ */
+'use strict';
+
+import {createSubCommand, extractRequiredInfo} from './command.js';
+import fetch from 'node-fetch';
+import {readFileSync, writeFileSync} from 'fs';
+import * as versionUtils from './versions.js';
+import JSON5 from 'json5';
+
+
+const DATABASE_URL = `https://raw.githubusercontent.com/google/blockly/develop/scripts/migration/renamings.json5`;
```

Comment:
are we sure this should be develop and not master? on the one hand i guess if changes are gated by the release you're updating to, it's fine to have unreleased changes here, but on the other hand develop is a lot more active and what if we break the renamings file during the quarter? then we've broken this script "in production" despite not doing a release

i also haven't read far enough to know how the default value of "to" version is chosen to know if that might be a problem from using develop branch